Modules/Quest/QuestieQuest.lua

```diff
@@ -444,7 +421,7 @@ function QuestieQuest:AbandonedQuest(questId)
 
         local quest = QuestieDB:GetQuest(questId);
         if quest then
-            quest.Objectives = {};
+            quest.Objectives = nil;
```

Comment:
I am not sure I like this change. I think it was just recently changed so a table, right? But an empty table is just as good as `nil` from the code logic perspective, but is also saver to not result in an error.